po/uk.po

```diff
@@ -767,26 +770,36 @@ msgid "Authentication is required to reset DNS settings."
 msgstr "Для скидання параметрів DNS до типових слід пройти розпізнавання."
 
 #: src/network/org.freedesktop.network1.policy:143
+msgid "DHCP server sends force renew message"
+msgstr "Сервер DHCP надсилає повідомлення щодо примусового оновлення"
+
+#: src/network/org.freedesktop.network1.policy:144
+#| msgid "Authentication is required to set a wall message"
```

Comment:
Misplaced comment?

Reply:
This is automated gettext fuzzy matching. Removed.